credit.py

```diff
@@ -36,3 +41,18 @@ def check_user_with_password_exists(self, name, password):
             return False
         return check_password_hash(self.load[name], password)
 
+def check_user_with_password_exists_sqldb(name, password):
+    db_session = sessionmaker(bind=create_engine(os.environ.get('TDA_DB'), echo=True))()
```

Comment:
`sessionmaker` produces a class for sessions and you create a session here. Could you create the `Session` class once and attach it to the app and then create sessions from it?